flexibleengine/acceptance/data_source_flexibleengine_waf_dedicated_instances_test.go

```diff
@@ -103,7 +103,8 @@ func testAccWafDedicatedInstances_conf(name string) string {
 %s
 
 data "flexibleengine_waf_dedicated_instances" "instance_1" {
-  name = flexibleengine_waf_dedicated_instance.instance_1.name
+  name 					= flexibleengine_waf_dedicated_instance.instance_1.name
+  enterprise_project_id = 0
```

Comment:
By default, `enterprise_project_id` is 0, so the `enterprise_project_id = 0` test case is invalid.

Reply:
If the resource do not support `enterprise_project_id` field, the unit test will be failed. If support, it will pass.